daemon/upgrademanager/upgrade.go

```diff
@@ -188,3 +189,20 @@ func upgradeFrom11(version int) {
 		go reProcessAllClaims()
 	}
 }
+
+func upgradeFrom12(version int) {
+	if version < 13 {
```

Comment:
just wondering, what happens if a previous upgrade fails? seems like the next upgrades would still try to get executed. You should use version == 12 instead, this ensures that you're upgrading from the right version only.

Reply:
if I do that then future upgrades will not happen. So if the data is at version 6 only the upgrade for version 6 will be applied. I actually need all upgrades to happen similar to database migrations. If only 1 ran and there were 2 that had to be run it would corrupt the data.

Reply:
well you'd have to update the version var from migration to migration